conbench/tests/populate_local_conbench.py

```diff
@@ -217,7 +217,7 @@ def register():
         "secret": "innocent-registration-key",
     }
     r = session.post(url, json=data)
-    if r.text and "Email address already in use":
+    if r.text and "Email address already in use" in r.text:
```

Comment:
Not sure how I even noticed this. But this was a bug that wanted to be fixed once I saw it.